website/src/pages/ga4gh/drs/v1/objects/[id]/index.ts

```diff
@@ -0,0 +1,240 @@
+
+    if (!result.isOk()) {
+        return new Response(
+            JSON.stringify({
+                status_code: 404,
+                msg: `Object not found: ${objectId}`,
+            }),
+            {
+                status: 404,
+
+                headers: {
+                    'Content-Type': 'application/json',
+                },
+            },
+        );
+    }
+
+    return new Response(JSON.stringify(result.value), {
+        status: 200,
+
+        headers: {
+            'Content-Type': 'application/json',
+        },
+    });
+};
+
+/**
+ * Get DRS object metadata for a sequence
+ */
+async function getDrsObject(objectId: string, origin: string): Promise<Result<DrsObject, ProblemDetail>> {
+    // Get sequence details from all configured organisms
+    const organisms = getConfiguredOrganisms();
+
+    // Parse the accessionVersion from the object_id
+    const { accession, version } = parseAccessionVersionFromString(objectId);
+
+    if (!accession || version === undefined) {
+        return err({
+            type: 'about:blank',
+            title: 'Invalid object ID',
+            detail: `Invalid object ID format: ${objectId}`,
+            status: 400,
+            instance: `/ga4gh/drs/v1/objects/${objectId}`,
+        });
+    }
+
+    // Try to get metadata from each organism until we find one
+    const promises = organisms.map(({ key }: { key: string }) =>
+        getObjectMetadata(objectId, key, origin).then((result) =>
+            result.isOk() ? ok(result.value) : Promise.reject(new Error(result.error.detail)),
+        ),
+    );
+
+    try {
+        const firstSuccess = await Promise.any(promises);
+
+        return firstSuccess;
+    } catch (_) {
+        return err({
+            type: 'about:blank',
+            title: 'Object not found',
+            detail: `Object not found: ${objectId}`,
+            status: 404,
+            instance: `/ga4gh/drs/v1/objects/${objectId}`,
+        });
+    }
+}
+
+/**
+ * Get sequence metadata for a specific organism and format it as a DRS object
+ */
+async function getObjectMetadata(
+    accessionVersion: string,
+    organism: string,
+    origin: string,
+): Promise<Result<DrsObject, ProblemDetail>> {
+    const lapisClient = LapisClient.createForOrganism(organism);
+
+    const detailsResult = await lapisClient.getSequenceEntryVersionDetails(accessionVersion);
+
+    if (!detailsResult.isOk()) {
+        return detailsResult as Result<never, ProblemDetail>;
+    }
+
+    // Get sequence data from the seq endpoint
+    const { accession, version } = parseAccessionVersionFromString(accessionVersion);
+    if (!accession || version === undefined) {
+        return err({
+            type: 'about:blank',
+            title: 'Invalid accession version',
+            detail: `Invalid accession version: ${accessionVersion}`,
+            status: 400,
+            instance: `/ga4gh/drs/v1/objects/${accessionVersion}`,
+        });
+    }
```

Comment:
I think this is a repeat of the check you do in getDrsObject - so I think it can be removed